packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -315,6 +312,22 @@ async function createLanguageClient(extensionContext: vscode.ExtensionContext) {
 
     languageClientUtils.setClientInstance(languageClient);
     await languageClient!.start();
+    // Client is running
+
+    // Listener is useful only in async mode
+    if (!languageServer.enableSyncInitJobs) {
+      // The listener should be set after languageClient is ready
+      // Language client will get notified once async init jobs are done
+      languageClient.onNotification(API.doneIndexing, async () => {
+        await getTestOutlineProvider().refresh();
+        languageServerReady();
+      });
+    } else {
+      // indexer must be running at the point
+      await getTestOutlineProvider().refresh();
+      languageServerReady();
```

Comment:
So previously we always waited on the ready notification to execute this logic. Now we'll be executing it prior to the doneIndexing event.  Won't this set the status to ready, then have it immediately set it back to indexing on line 335 below? 

It seems like this piece of the logic should go below setting the status to indexing.